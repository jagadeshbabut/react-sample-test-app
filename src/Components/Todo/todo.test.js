import React from 'react';
import {shallow} from 'enzyme';
import Todo from './todo';

describe('Todo Component', () => {
    it('Renders without crashing', () => {
        shallow(<Todo/>)
    });

    it('Todo component has not changed', () => {
        const wrapper = shallow(<Todo/>);
        expect(wrapper).toMatchSnapshot()
    });

    it('Check Goto Todos link is working', () => {
        const wrapper = shallow(<Todo/>);
        // expect(wrapper).
    });

});