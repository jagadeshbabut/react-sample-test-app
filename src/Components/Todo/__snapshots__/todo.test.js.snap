// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo Component Todo component has not changed 1`] = `
<div>
  <form
    className="App"
    onSubmit={[Function]}
  >
    <input
      onChange={[Function]}
      value=""
    />
    <button>
      Submit
    </button>
  </form>
  <List
    items={Array []}
  />
</div>
`;
